src/AnalysisDriver.h

```diff
@@ -0,0 +1,19 @@
+#ifndef __ANALYSIS_DRIVER_H__
+#define __ANALYSIS_DRIVER_H__
+
+#include "Configuration.h"
+#include "EvalExpressionAnalysis.h"
+#include <memory>
+
+class AnalysisDriver {
+
+  public:
+    AnalysisDriver(const Configuration &configuration);
+    void closure_entry(const SEXP call, const SEXP op, const SEXP rho);
+
+  private:
+    Configuration configuration_;
+    std::unique_ptr<EvalExpressionAnalysis> eval_expression_analysis_;
```

Comment:
shall this be just `EvalExpressionAnalysis eval_expression_analysis_`?

Reply:
Yes, this will be updated.